Purchases/Purchasing/OfferingsManager.swift

```diff
@@ -117,7 +117,7 @@ private extension OfferingsManager {
                 result[product.productIdentifier] = product
             }
 
-            if let createdOfferings = self.offeringsFactory.createOfferings(fromProductDetailsByID: productsByID,
+            if let createdOfferings = self.offeringsFactory.createOfferings(fromStoreProductsByID: productsByID,
```

Comment:
this is probably swiftier as `self.offeringsFactory.createOfferings(from:`, but I'm trying to restrict this PR to only this rename